### ISSUE TITLE
Script greek style incongruous with serifed roman glyphs

### ISSUE THREAD

**Issue by katef** [skosch/Crimson#19]
For Crimson-Roman, the greek drawn with a rounded, handwritten script style. These look lovely, of course, but don't go with the style for the rest of the font. I presume this is an intentional design decision, since I can't find any alternate greek glyphs in the source. What's the rationale for these?

Here for example, I would expect the capital lambda to have the same style as the capital A.
![ABΛΒ](http://dioptre.org/tmp/crimson-roman-greek.png)

My suggestion is to have the greek glyphs in serifed style by default, and to move the existing glyphs to stylistic alternates instead.

I realise that drawing new glyphs is quite a lot of work. TeX Gyre's Pagella has quite similar greek glyphs; perhaps some could be borrowed from there, and tweaked to fit the same proportions as for Crimson's existing script-style glyphs.

**Comment by skosch** [comment]
Yeah, that was a deliberate (if difficult) decision. I figured I'd design the Greek to be a real text face, math looks be damned. I've always thought latinized, chiselled-style Greek to look somehow *wrong* at least in text. I particularly liked what Slimbach did in his Garamond Premier Pro but wanted the uppercase ductus to match the lowercase. I did do quite a bit of research and experimentation before settling on this (see e.g. CatharsisFonts/Cormorant#6).

Aware that it'd clash with the Roman, I believed that at some point I'd create a Real Math Font, including a Greek with a Latin look, at [U+1D6A8](https://en.wikipedia.org/wiki/Mathematical_operators_and_symbols_in_Unicode#Mathematical_Alphanumeric_Symbols_block) ... except I never got around to it.

In the meantime, you could absolutely mix & match fonts, though. Have you checked out the [Greek Font Society](http://www.greekfontsociety.gr/pages/en_typefaces1.html) yet?

**Comment by katef** [comment]
@skosch Okay - Would you be okay with adding stylistic alternates for chiselled style greek? Then I could specify the opentype feature for that, and I'd be a happy bunny.

Just to clarify, I'm not talking about anything specifically for maths, here.

**Comment by skosch** [comment]
Oh I would be totally okay, and I appreciate the offer, I just don't have the time these days.

**Comment by katef** [comment]
Great! Do you know where they should live? Somewhere in the private use area?

**Comment by skosch** [comment]
I think so? Although come to think of it, why *not* put it in the math section? Isn't that just the point?

**Comment by katef** [comment]
I don't think this should go in the math section because it's not specific to mathematical use. E.g. I wouldn't want to use mathematical codepoints to write somebody's name.

Although perhaps the math section could be populated with references to these new glyphs.

**Comment by skosch** [comment]
Well, I'll leave it up to you then. References in the math section would make sense.

I'd prefer to leave the current Greek untouched – *unless* you want to create six weights of Latinesque Greek, then we could talk about switching out the current calligraphic glyphs as alternates.

**Comment by katef** [comment]
Yes, I'm not signing up for that just yet :) I'm only intending to work on Roman for the moment, and probably just uppercase at first. Of course I agree about leaving the current glyphs where they are.
